components/rowListScene.brs

```diff
@@ -1,12 +1,21 @@
 sub init()
-    m.global.addField("http", "node", FALSE)
     m.global.addField("ratio", "float", FALSE)
-    m.global.http = createObject("roSGNode", "httpTask")
-    m.global.http.control = "RUN"
+    m.flag = true
     m.global.ratio = 1
     m.rowList = m.top.findNode("mainRowList")
+    m.MainBoldLabel = m.top.findNode("MainBoldLabel")
+    m.DateAndDurationLabel = m.top.findNode("DateAndDurationLabel")
+    m.DescriptionLabel = m.top.findNode("DescriptionLabel")
+    m.CategoryLabel = m.top.findNode("CategoryLabel")
```

Comment:
To increase `findNode` performance 
```suggestion
    m.heroDetailsId = m.top.findNode("heroDetailsId")
    m.MainBoldLabel = m.heroDetailsId.findNode("MainBoldLabel")
    m.DateAndDurationLabel = m.heroDetailsId.findNode("DateAndDurationLabel")
    m.DescriptionLabel = m.heroDetailsId.findNode("DescriptionLabel")
    m.CategoryLabel = m.heroDetailsId.findNode("CategoryLabel")
```

Reply:
To not make start from top all over again, that's amazing to know !

Reply:
Done